roles/slurm/templates/configure_slurm_accounting_db.bash

```diff
@@ -31,7 +32,7 @@ sacctmgr -i modify qos Name='leftover' set \
     Priority=0 \
     UsageFactor=0 \
     GrpSubmit=30000 MaxSubmitJobsPU=10000 \
-    GrpTRES=cpu=0,mem=0,gres/gpu=0
+    GrpTRES={% if slurm_cluster_gpus_total | int > 0 %}gres/gpu=0,{% endif %}cpu=0,mem=0
```

Comment:
I am not sure that this change add anything (except expects athe value to be set, which is, but still).
It's fine the way you wrote it down, but I don't think this `if` is really needed. I tested on non-gpu clusters, and the default value of 'gres/gpu=0' is perfectly fine.

Reply:
I could not run the script on a non GPU cluster like Winged Helix; Got an error that the gres/gpu type TRES did not exist. I believe this is due to this 
```
{% if slurm_cluster_gpus_total | int > 0 %}GresTypes=gpu
AccountingStorageTRES=gres/gpu
{% endif %}
```
in the `roles/slurm/templates/slurm.conf.22.05.2-1.el7.umcg` template. The alternative would be to remove the if statement there too and always specify both `GresTypes=gpu` and `AccountingStorageTRES=gres/gpu` even if the number of available GPUs is 0.

Reply:
It did not fail on other tests, like on Hyperchicken. I also redeployed the slurm on Talos and it did not fail there... The only reason where that is different it that stale `/etc/slurm/gres.conf` file was solving HC, while failing on WH. But that's weird.
I will further investigate the existing slurm role on another non-gpu machine...